desktop/electron-builder.yml

```diff
@@ -20,7 +20,7 @@ extraResources:
       - "**/*"
   - "ThirdPartyNotices.txt"
   - "data"
-  - "resources"
+  - "src/generated/localization"
```

Comment:
This makes me think we should stick with `resources/*` or something similar for non-Typescript/Javascript files. I don't think we want our distributed packages to have a `src` directory which only contains `src/generated/localization` (which would include some Typescript files that may not compile). Let's follow up and talk about this, since I think we also need to differentiate between resources under source control (like *.css files, etc) and ones that are output artifacts of the build.

Reply:
Changed file structure so we're back to having the JSON/REJSON files in the `resources` directory and Typescript file in `generated/localization`. Also changed directory hierarchy to be `generated/localization` instead of `localization/generated` like it was previously